### DISCUSSION TITLE
How to disable HTTP2

### DISCUSSION THREAD

**Discussion by ghost** [kubernetes-client/csharp#1144] [Q&A] [answered]
Hi Team, 

We want to disable http2 protocol. Is there any way to do it. I can see configurations in code but not available in latest nuget package. 
Can someone guide on this ?

**Comment by tg123** [comment, marked as answer]
will publish nuget  soon

**Reply by ghost** [reply]
Any tentative date or timeline for this ?

**Reply by tg123** [reply]
https://www.nuget.org/packages/KubernetesClient/10.0.1
